### ISSUE TITLE
Build broken on latest nightly

### ISSUE THREAD

**Issue by paolobarbolini** [fusion-engineering/inline-python#61]
I'm not sure what happened but the latest nightlies don't compile anymore.

```
error[E0432]: unresolved import `proc_macro::LineColumn`
 --> macros/src/embed_python.rs:1:18
  |
1 | use proc_macro::{LineColumn, Span};
  |                  ^^^^^^^^^^ no `LineColumn` in the root
  |
  = help: consider importing this struct instead:
          proc_macro2::LineColumn

error[E0615]: attempted to take value of method `column` on type `proc_macro::Span`
  --> macros/src/embed_python.rs:69:14
   |
69 |                     end_loc.column = end_loc.column.saturating_sub(end.len());
   |                             ^^^^^^ method, not a field
   |
   = help: methods are immutable and cannot be assigned to

error[E0615]: attempted to take value of method `column` on type `proc_macro::Span`
  --> macros/src/embed_python.rs:69:31
   |
69 |                     end_loc.column = end_loc.column.saturating_sub(end.len());
   |                                              ^^^^^^ method, not a field
   |
help: use parentheses to call the method
   |
69 |                     end_loc.column = end_loc.column().saturating_sub(end.len());
   |                                                    ++

error[E0615]: attempted to take value of method `line` on type `proc_macro::Span`
  --> macros/src/error.rs:56:35
   |
56 |         .skip_while(|span| span.start().line < line)
   |                                         ^^^^ method, not a field
   |
help: use parentheses to call the method
   |
56 |         .skip_while(|span| span.start().line() < line)
   |                                             ++

error[E0615]: attempted to take value of method `line` on type `proc_macro::Span`
  --> macros/src/error.rs:57:35
   |
57 |         .take_while(|span| span.start().line == line);
   |                                         ^^^^ method, not a field
   |
help: use parentheses to call the method
   |
57 |         .take_while(|span| span.start().line() == line);
   |                                             ++

Some errors have detailed explanations: E0432, E0615.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `inline-python-macros` (lib) due to 5 previous errors
```

**Comment by n3vu0r** [comment]
Should be fixed by #60.

**Comment by m-ou-se** [comment]
Fixed in 0.12!
